packaging/bundle/scripts/windows/bundle.ps1

```diff
@@ -74,20 +74,28 @@ function install_python([string]$buildDir=$BUILD_DIR, [string]$pythonVersion=$PY
 
     if (((& $nugetPath sources list 2> $null) | Select-String "nuget.org") -Eq $null) {
         & $nugetPath sources add -name "nuget.org" -source "https://api.nuget.org/v3/index.json"
+        if ($lastexitcode -ne 0){ throw }
     }
 
     & $nugetPath install python -Version $pythonVersion -OutputDirectory $buildDir
-    mv "$installPath\tools" $targetPath
-
+    if ($lastexitcode -ne 0){ throw }
+    # Recursively copy of installPath\tools to targetPath
+    New-Item -ItemType Directory -Path $targetPath | Out-Null
```

Comment:
The `mv` was causing frequent troubles in my local tests, this more verbose solution still had issues but much less frequently. Likely some tool or even docker itself taking time to allow the file deletion.